backend/main/test/Group/MockContainerRepository.ts

```diff
@@ -0,0 +1,33 @@
+import { IContainerRepository } from "src/Group/Domain/IContainerRepository";
+import { Container, ContainerId } from "src/Group/Domain/Entities/Container";
+
+export class MockContainerRepository implements IContainerRepository {
+	memoryContainers: Container[] = [];
+
+	async find(id: ContainerId): Promise<Container | null> {
+		return (
+			this.memoryContainers.find((container) => {
+				container.id.equal(id);
```

Comment:
You must return true to return the value.

```ts
if (container.id.equal(id)) return true;
```
or

remove the {} braket.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find